ocean/jigsaw_to_MPAS/build_mesh.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
```

Comment:
I noticed in the process of rebasing that this PR reintroduces this file.  We no longer use the COMPASS version, we get `build_mesh` from `mpas_tools`.  This needs to be removed.  I'll try to comment wherever it might be getting used.

Reply:
It doesn't look like you're using this, so it just needs to be deleted.  Since it's my branch, I guess I'll take care of it.